_posts/2017-01-30-inc-cmd-lessonslearned.md

```diff
@@ -0,0 +1,79 @@
+
+#### Understand battle rhythm
+An IC is on a constant running battle between triaging the problem, tasking resources, communicating with customers and stakeholders and considering contingency plans. Regularly recap the situation for the people on the conference call or Slack channel to focus them, this can be done with a CAN report to keep things in a 'standard' format so everyone understands what is going on.
+
+-	Conditions – Current status
+-	Actions – What is being done or needs to be done?
+-	Needs – What resources are needed or other action items required?
+
+**Note:** Identify who you are talking to at the start of your sentence to get their attention, and use pauses wisely to allow people to digest what you are communicating
+
+For example:
+
+> **C:** “IC: Okay everyone listen in. Let’s take a look at where we are with this incident. Incident has been open for 30 minutes situation as I understand it is that we have narrowed down the problem to the DC firewall blocking access to the International office.”
+
+> **A:** “IC: Network ops have looked at it and are 90% certain that it is a problem with a runaway process on the firewall. They are currently working to fail over the cluster to resolve this within the next 10 minutes.”
+
+> **N:** “IC: I want to keep everyone on the conf call for now until we are certain of the fix. While we are waiting, lets discuss our next most likely root cause candidate and what our plan B will be.”
+
+#### On that subject, have a plan B!
+When your team have come up with plan A and are focussing on that you need to be thinking about plan B and C, getting resource lined up to support them and switching assets to support a secondary or tertiary plan when appropriate in a timely manner. Do not be afraid to adapt or change your plans where it is justified to do so. Your main focus should always be on restoring service as quickly as possible and it can sometimes be a tough call whether to ask a team to 'switch fire' to another plan. Where possible have multiple teams working on different plans rather than having a single team context switching to come up with multiple plans.
+
+#### Don’t forget the customer!
+Our Service Lifecycle Managers are our IC's interface into our service operations and the customer facing teams, if you don't have this luxury then you can nominate someone to regularly 'check in' with the business and customer stakeholders and make sure you know whether your customers are contacting you.
+
+Don't forget you can use proactively use both technical and non-technical resources such as maintenance pages or/outage messaging systems and issue either internal or external communications which may help a potential problem from getting worse. Good external communications help your customers understand what is going on without them contacting your customer service teams en masse to find out why their account isn't working!
```

Comment:
"en masse" - generally accepted semantic HTML usage for this kind of thing these days is `<i lang="fr">en masse</i>`. Happy to be corrected by any i18n / a11y experts though.